src/Compilers/CSharp/Portable/Symbols/Tuples/TupleFieldSymbol.cs

```diff
@@ -277,16 +262,17 @@ internal sealed class TupleVirtualElementFieldSymbol : TupleElementFieldSymbol
             ImmutableArray<Location> locations,
             bool cannotUse,
             bool isImplicitlyDeclared,
-            TupleElementFieldSymbol? correspondingDefaultFieldOpt)
+            FieldSymbol? correspondingDefaultFieldOpt)
```

Comment:
>correspondingDefaultFieldOpt [](start = 25, length = 28)

Can we assert is not a ```TupleElementFieldSymbol```?

Reply:
This could be a `TupleElementFieldSymbol` in `(int Alice, int Bob)` scenario, where the `.Item1` is a `TupleElementFieldSymbol`. Per our discussion, I'll expand test to show that field.

It could be `TupleVirtualElementFieldSymbol` in `(int Alice, ..., int Hannah)` scenario, where the `.Item8` is itself a `TupleVirtualElementFieldSymbol`.

---
In reply to: [426830948](https://github.com/dotnet/roslyn/pull/44231#discussion_r426830948) [](ancestors = 426830948)